packages/components/src/VirtualizedList/RendererSelector.component.js

```diff
@@ -92,17 +93,19 @@ class RendererSelector extends React.Component {
 			};
 		} else {
 			ListRenderer = ListGrid;
-			customProps = { rowRenderer: getRowRenderer(type) };
+			customProps = { rowRenderer: getRowRenderer(type, this.props.rowRenderers) };
```

Comment:
You are only allowing custom stuff when not in table mode. Later on we'll be nice to let custom the table mode nop ?

Reply:
The table can already be customized at the cell level AFAIK